### DISCUSSION TITLE
Rust connection url example is incorrect

### DISCUSSION THREAD

**Discussion by oscartbeaumont** [planetscale/discussion#278] [Bug Reports] [answered]
When getting the connection URL from Planetscale it shows snippets from a bunch of different programming languages to help get you started. The Rust examples shows putting the connection URL in a `.env` file and then also adding `dotenv` as a dependency however the `dotenv.ok();` line is never shown in the `src/main.rs`. This line is required so the environment variable is read from the `.env` file.

Fix by changing `src/main.rs` to:
```rust
use std::env;
use dotenv::dotenv;

fn main() {
    dotenv.ok();
    let url = env::var("DATABASE_URL").expect("DATABASE_URL not found");
    let builder = mysql::OptsBuilder::from_opts(mysql::Opts::from_url(&url).unwrap());
    let pool = mysql::Pool::new(builder.ssl_opts(mysql::SslOpts::default())).unwrap();
    let _conn = pool.get_conn().unwrap();
    println!("Successfully connected to PlanetScale!");
}
```

**Comment by fooforge** [comment, marked as answer]
Hey @oscartbeaumont,

Thanks for letting us know! I will fix this up shortly.

**Reply by AuroraLantean** [reply]
Hi is Rust working with PlantScale?

**Reply by mscoutermarsh** [reply]
@AuroraLantean Yes!
